src/landscape/components/LandscapeForm/Actions.js

```diff
@@ -17,8 +35,16 @@ const Actions = props => {
           {t('landscape.form_back')}
         </Button>
         <Stack direction={{ xs: 'column', sm: 'row' }} spacing={1}>
-          <Button variant="outlined">{t('landscape.form_save_now')}</Button>
-          <Button variant="contained">{t('landscape.form_next')}</Button>
+          <Button
+            variant="outlined"
+            onClick={onSaveWrapper}
+            sx={{ pl: 2, pr: 2 }}
+          >
+            {t('landscape.form_save_now')}
+          </Button>
+          <Button variant="contained" onClick={onNext} sx={{ pl: 6, pr: 6 }}>
```

Comment:
@david-code those are just abbreviations for paddingLeft and paddingRight, you also find those for margins, I think since those a really common to use they decided to create these more compact representations